### ISSUE TITLE
Random comment

### ISSUE THREAD

**Issue by anthropomorphic** [jakeleveroni/Weather-App#1]
Dude, props on having super readable code. Like, damn. I never write comments on my code and when I go back and look at it I can't understand a thing. Your code takes like no time to understand. `billions_of_comments == AWESOME`.

**Comment by jakeleveroni** [comment]
Hey man thanks yeah i started this a while ago i was going to make an app for weather and traffic for my dad to run on a raspberry-pi so he could just plug it into a tv and have it do all the work for him. But life picked up again and I have not been able to get around to it in a while. Thanks for the compliment I'm kind of neurotic about comments for some reason.

**Comment by anthropomorphic** [comment]
Haha keep it up. I used to be neurotic about abstraction, and while I've since learned to curb my desire to write **absolutely all** of my code so that it could _theoretically someday_ be used in some other context (which meant I took **forever** to get anything done), writing all that super generic code taught me a ton. Neuroticism is a programmer's best friend =)
Also thats kind of a neat idea btw. I've been planning on getting a pi for a while now, but I haven't bought one yet cause I haven't thought of anything specific to do with it lol.
